src/main/java/backend/Logic.java

```diff
@@ -218,7 +218,7 @@ public void replaceIssueLabelsUI(TurboIssue issue, List<String> labels) {
      * and then sends the data to the GUI.
      */
     private void refreshUI() {
-        updateController.filterSortRefresh(getAllUIFilters());
+        updateController.filterSortCountRefresh(getAllUIFilters());
```

Comment:
Hmm, I don't think we need to change this method name -- `count` is kind of like a filter. It can't keep growing as we add more features.